pom.xml

```diff
@@ -7,10 +7,10 @@
   <groupId>org.eclipse.tracecompass</groupId>
   <artifactId>trace-event-logger</artifactId>
   <version>0.0.3-SNAPSHOT</version>
-
+  <packaging>jar</packaging>
   <name>trace-event-logger</name>
   <!-- the project's website -->
-  <url>http://www.tracecompass.org</url>
+  <url>https://eclipse.dev/tracecompass</url>
```

Comment:
Should that be the URL of the repo?

Reply:
I think that would be the <scm> tag. not sure.

Reply:
Ok